src/pages/RegisterStudents.vue

```diff
@@ -14,7 +14,7 @@
               <li>username</li>
               <li>date of birth</li>
               <li>grade</li>
-              <li>password</li>
+              <li>password (min 6 characters)</li>
```

Comment:
Is this change intentional? Just checking as it seems unrelated to the PrimeVue update